themes/sensei-course-theme/readme.txt

```diff
@@ -1,8 +1,8 @@
 === Sensei LMS ===
 Contributors: Automattic
-Requires at least: 5.7
```

Comment:
It seems it was untouched for a long time, but it's probably not very important.

Reply:
Is this something that's supposed to be bumped every time?